README.md

```
@@ -22,7 +22,7 @@
 - [Current Status](#current-status)
 - [What's New](#whats-new)
 - [Highlights](#highlights)
-- [Feature Status Matrix](#feature-status-matrix)
+ - [Feature Status](#feature-status)
```

markdownlint-cli2 (0.18.1)

Inconsistent indentation for list items at the same level
Expected: 0; Actual: 1

(MD005, list-indent)